src/interfaces/ISAFEEngine.sol

```diff
@@ -123,23 +123,23 @@ interface ISAFEEngine is IAuthorizable, IDisableable, IModifiable, IModifiablePe
   // --- Errors ---
 
   /// @notice Throws when trying to modify parameters of an uninitialized collateral type
-  error SAFEEng_CollateralTypeNotInitialized();
+  error SAFEEng_CollateralTypeNotInitialized(); // 0xcc8fbb29
```

Comment:
Are these here to help with debugging, since the actual error message is not shown?

Reply:
yeah, the function selector is the first 4-bytes of the keccak hash of the name:
keccak256(SAFEEng_CollateralTypeNotInitialized()) = 0xcc8fbb29....